### DISCUSSION TITLE
[@types/sap__approuter] typo in definition

### DISCUSSION THREAD

**Discussion by willempardaens** [DefinitelyTyped/DefinitelyTyped#70120] [Issues with a @types package]
The property name `xsAppConfig` in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/sap__approuter/index.d.ts#L122 does not match `xsappConfig` (lowercase 'A') from the implementation schema definition in "/node_modules/@sap/approuter/lib/configuration/schemas/options-schema.json" line 31:

```
xsAppConfig?: ComSapXsappSchema_82;
```

```
"xsappConfig": { "type": "object" }
```

Using @sap/approuter v16.6.0

[a comment by an automated account is not shown]

**Comment by nbblum** [comment]
Hi, thanks for the hint I will look into it

**Reply by nbblum** [reply]
I opened a PR to fix it #70161

**Reply by nbblum** [reply]
The definition is now up to date with the schema

**Comment by willempardaens** [comment]
Thanks for the quick action!
